### ISSUE TITLE
call service function from template without watching !

### ISSUE THREAD

**Issue by amineparis** [angular/angular#22084]
When we make a call to service function directly from template, angular keeps calling the function infinitely which affects application performance. Is their a way to call it only once ?

**Comment by trotyl** [comment]
Duplicate of #14033, #16777. And it's support question.

**Comment by amineparis** [comment]
I don't think it is called binding what I talk about here. I talk about service function calls from templates

**Comment by ericmartinezr** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).

<hr>

Don't use a function. Assign the value the to a property in your component.

**Comment by amineparis** [comment]
@ericmartinezr this may be an improvement on Angular to make it call only once as default behaiviour

**Comment by trotyl** [comment]
"Rendering without watching" is called "one-time binding", that's the conversion name used for years.

**Comment by mhevery** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).
